docs/links/podman-image-rm.1

```diff
@@ -1 +1 @@
-.so man1/podman-rm.1
+.so man1/podman-rmi.1
```

Comment:
I don't think we should do this change?  Is there a reason why you think it's needed?

Reply:
That looks correct to me.
`podman-image-rm` is the same as `podman rmi`
`podman-container-rm is the same as `podman rm`

Reply:
`man podman-image-rm` gives the manpage for podman-rm (podman-container-rm)

Reply:
Thanks, my dyslexia must have been kicking in, never saw the image part of that.